tools.txt

```diff
@@ -6,7 +6,7 @@ and=und
 Anonymize diagnostics=Diagnostik anonymisieren
 Anytime=Jederzeit
 Archived Notifications=Archivierte Benachrichtigungen
-Are you sure=Bist du sicher
+Are you sure=Sind Sie sicher
```

Comment:
Wieso du? Man kann ja nicht pauschal vom duzen ausgehen?!

Reply:
Sorry, was? Ich verstehe nicht ganz, was du meinst. Wenn du es dir genau anschaust, dann siehst du, dass ich "Bist du sicher" zu "Sind Sie sicher" geändert habe.

Reply:
Let me know if there is a discrepancy or if you want me to merge this PR. Thx

Reply:
I would like you to merge this PR

Reply:
You got it. Cheers